pkg/cmd/create/create_cluster_eks.go

```diff
@@ -227,7 +227,5 @@ cluster provisioning. Cleaning up stack %s and recreating it with eksctl.`,
 	o.InstallOptions.SetInstallValues(map[string]string{
```

Comment:
There is no reason to set the install values here since the install doesn't get call anymore after create cluster command.